main.tex

```diff
@@ -379,31 +384,39 @@ \section{Models}
 measured with a single spectrum.  \pyspeckit provides the \texttt{hyperfine}
 model class to handle this class of molecular line transitions, and it includes
 several molecular species implementations (HCN, N$_2$H$^+$, NH$_3$,
-H$_2$CO). {\color{red}If this is the beefiest part of pyspeckit, we should include a table
-listing the models and potentially an example for how to customize a model?}
+H$_2$CO). Models can customizable and examples of registering a new or modified
+model in \pyspeckit are included in the online documentation.
 
 \section{Cubes}
 \label{sec:cubes}
 Spectral cubes are growing more important in radio astronomy since they are the
 natural data products produced by interferometers like ALMA and the JVLA.
-Optical and infrared data cubes from integral field units (IFUs) like MUSE on
-the VLT, OSIRIS on Keck,  NIFS on Gemini, and NIRSpec on JWST are also becoming
-more common.
+Optical and infrared data cubes are also becoming
+more common from integral field units (IFUs) like MUSE on
+the VLT, OSIRIS on Keck, NIFS on Gemini, and NIRSpec and MIRI on JWST.
 
 While many cube operations are handled well by \texttt{numpy}-based packages
 like \texttt{spectral-cube}\footnote{\url{spectral-cube.readthedocs.io}},
-it is sometimes necessary or desirable to fit a profile to each spectrum
+it is sometimes desirable to fit a profile to each spectrum
 in a cube.  The \texttt{Cube.fiteach} method is a tool for automated line
 fitting that includes parallelization of the fit. Implementation examples can be
-found in the online documentation. This tool  has  seen significant use in
+found in the online documentation. This tool  has seen significant use in
 papers and been incorporated into other tools (e.g.,
 \texttt{multicube}\footnote{\url{https://github.com/vlas-sokolov/multicube}}).
 
 \section{Summary}
-\texttt{pyspeckit} is a versatile tool for spectroscopic analysis.
+\texttt{pyspeckit} is a versatile tool for spectroscopic analysis in python and is
+one of the \astropy affiliated packages. \pyspeckit can interactively fit a model 
+to a spectrum using the Levenberg-Marquardt optimization method via \texttt{mpfit} and 
+\texttt{lmfit}, and wrappers for \texttt{pymc} and \texttt{emcee} are also available.
+There is also the option to non-interactively fit a model to the many spectra in a
```

Comment:
don't specify non-interactively here; non-interactive works for both individual spectra and cubes